docs/content/en/docs/workflows/ci-cd.md

```diff
@@ -4,20 +4,26 @@ linkTitle: "Continuous Delivery"
 weight: 40
 ---
 
-Skaffold offers several sub-commands for its workflows that make it quite flexible when integrating with CI/CD pipelines.
+Skaffold provides several features and sub-command "building blocks" that make it very useful for integrating with (or creating entirely new) CI/CD pipelines.
+The ability to use the same skaffold.yaml for iterative development and continuous delivery make handing off an application from a development team to an ops team a breeze!
 
+Let's start with the simplest use case: a single, full deployment of your application.
 
 ## `skaffold run`
+{{< maturity "run" >}}
 
-`skaffold run` is a single command for a one-off deployment. It includes all the following phases as it builds, tags, deploys and waits for the deployment to succeed if specified.
-We recommend `skaffold run` for a simple Continuous Delivery setup, where it is sufficient to have a single step that deploys from version control to a cluster.
-For more sophisticated Continuous Delivery pipelines, Skaffold offers building blocks that are described next:
+`skaffold run` is a single command for a one-off deployment. It runs through every major phase of the Skaffold lifecycle: building your application images, tagging these images (and optionally pushing them to a remote registry), deploying your application to the target cluster, and monitoring the created resources for readiness.
```

Comment:
Would be worth pulling the lifecycle phase descriptions to the top so that they can be referenced by name here.

Reply:
i'm keeping a running list of improvements we should make to these docs, but they'll get addressed outside of this PR